src/test/java/net/snowflake/ingest/streaming/internal/DataValidationUtilTest.java

```diff
@@ -269,6 +270,7 @@ public void testValidateAndParseTimestamp() throws ParseException {
 
     // Test integer-stored time and scale guessing
     SimpleDateFormat df = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS");
+    df.setTimeZone(TimeZone.getTimeZone("UTC"));
```

Comment:
This is interesting, is this a new behavior change in JDBC?

Reply:
the tests were not passing because of various TZ, I doubt it has anything to do with JDBC. 
I ran it on mac, someone else might run on devvm?

Reply:
ah, might be the case, i use devvm, merge gate all passed without this line as well